refresh.py

```
@@ -0,0 +1,223 @@
+                console.print(f"[WARN] Unknown config key '{key}'.")
+    # Password warning
+    password = cfg.get("backup_password")
+    if password and len(password) < 4:
+        console.print("[WARN] Backup password is very short; consider using a longer password.")
+
+# ----------------- Commands -----------------
+def run_command(cmd, capture=False):
+    if CONFIG.get("dry_run"):
+        console.print(f"[dry-run] {cmd}")
+        return "" if not capture else None
+    result = subprocess.run(cmd, shell=True, capture_output=True, text=True)
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)